snippets/Responsive-Nuemorphism-Card-Design.md

```diff
@@ -0,0 +1,176 @@
+
+---
+title: Responsive Nuemorphism Card Design
+tags: interactivity,layout
+---
+
+Creates an awesome nuemorphism effect with cool interactivity to the users. 
+
+- Uses `display: flex` property along with `justify-content: center; align-items: center;` to make the snippet responsive.
+- Creates Nuemorphism effect by using nuemorphism colors like `rgba(255, 255, 255, 0.1)` or `rgba(0, 0, 0, 0.6)` and using nuemorphism shadows. 
+- Uses `:hover` pseudo-selector to make the transitions and interactivity.
+
+```html
+<div class="container">
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>01</h2>
+        <h3>Card One</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>02</h2>
+        <h3>Card Two</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>03</h2>
+        <h3>Card Three</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+</div>
+```
+
+```css
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700;800&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  min-height: 100vh;
+  background: #232427;
+}
+
+body .container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  max-width: 1200px;
+  margin: 40px 0;
+}
+
+body .container .card {
+  position: relative;
+  min-width: 320px;
+  height: 440px;
+  box-shadow: inset 5px 5px 5px rgba(0, 0, 0, 0.2),
+    inset -5px -5px 15px rgba(255, 255, 255, 0.1),
+    5px 5px 15px rgba(0, 0, 0, 0.3), -5px -5px 15px rgba(255, 255, 255, 0.1);
+  border-radius: 15px;
+  margin: 30px;
+  transition: 0.5s;
+}
+
+body .container .card:nth-child(1) .box .content a {
```

Comment:
These selectors are very high in specificity, I think we could survive with just targeting the class or element as necessary.